TicTacToe/TicTacToe.js

```diff
@@ -0,0 +1,157 @@
+class Game {
+    moves = 0;
+    user = 'X';
+    board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+    messages = "Game - START!";
+    complete = false;
+
+    reset() {
+        this.moves = 0;
+        this.user = 'X';
+        this.board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+        this.messages = "Game - START!";
+        this.complete = false;
+    }
+
+    getMoves() {
+        return this.moves;
+    };
+
+    displayBoard() {
+        let text = '';
+        for (let i = 0; i < 3; i++) {
+            for (let k = 0; k < 3; k++) {
+                text += " " + this.board[i][k];
+            }
+            text += "\n";
+        }
+        console.log(text);
+        return text;
+    };
+
+    getMessages() {
+        return this.messages;
+    }
+
+    makeMove(x, y) {
```

Comment:
What is this function supposed to return? I would say document each function here with a brief description, especially pointing out nontrivial input parameters and return values. (You don't have to document trivial getters/setters).

Note that this function sometimes returns nothing, and sometimes returns a string. This is generally not a good idea in a typed system (unless you're dealing with explicitly `Optional` return values or dealing with a dependent typing system).